content/index.md

```diff
@@ -7,3 +7,5 @@ title: Global Digital Library
 [Browse APIs](/browse)
 
 [Example](/example)
+
+[Content Guidelines](/example)
```

Comment:
@christer-io Kan du ikke bare rename "Example" til "Content Guidelines" siden de linker til samme side? 🙂 Eller så kan du endre `[Content Guidelines](/example)` til feks `[Content Guidelines](/guidelines)`.

Reply:
Loving it, jeg legger til en markdown-fil og lager en /guidelines. Mekker på det ikveld.